platform/viewer/src/components/ViewportGrid.tsx

```diff
@@ -54,20 +55,35 @@ function ViewerViewportGrid(props) {
         return;
       }
 
-      // Match each viewport individually
-      const numViewports = viewportGrid.numRows * viewportGrid.numCols;
+      const displaySetsInGrid = [];
+      const blankViewportIndices = [];
+
+      // Match each viewport individually.
+      const numViewports = viewportGridService.getNumViewportPanes();
+
       for (
         let viewportIndex = 0;
         viewportIndex < numViewports;
         viewportIndex++
       ) {
+        const displaySetsInViewport =
```

Comment:
these are displaySetInstanceUIDs and not displaySets.

Reply:
samme of diplsyaSetsInGrid

Reply:
`viewportDisplaysetInstanceUIDs`?

Reply:
I will go with the latter.